modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -557,6 +557,15 @@
 
 /// Handles weapon reagent imbuing
 /obj/structure/reagent_forge/proc/handle_weapon_imbue(obj/attacking_item, mob/living/user)
+	//This code will refuse all none ashwalkers from imbuing
+	if(!ishuman(user))
+		to_chat(user, span_danger("[attacking_item] shakes-- it refuses to be imbued!"))
+		return
+	var/mob/living/carbon/human/human_user = user
+	if(!istype(human_user.dna.species, /datum/species/lizard/ashwalker))
```

Comment:
Don't we have like an `isashwalker()` macro?

Reply:
not to my knowledge.

Reply:
```suggestion
	if(!is_species(human, /datum/species/lizard/ashwalker))
```
Or add an `is_ashwalker` macro somewhere, which should just be `is_species(human, /datum/species/lizard/ashwalker)`.

Reply:
I would, but that would need to be atomized-- so not for this PR atm.

Reply:
Then just use the `is_species` version and I can get it in my PR that adds `isashwalker`.